.github/workflows/stale.yml

```diff
@@ -48,10 +48,10 @@ jobs:
             This PR is being marked as stale since it has not had any activity in the last 180 days. 
             If you would like to keep this PR alive, please leave a comment asking for a review. 
             If the PR has merge conflicts, update it with the latest from the base branch.
-            <p>
+
             If you are having difficulty finding a reviewer, please reach out to the 
             [community](https://flink.apache.org/what-is-flink/community/).
```

Comment:
one other thing I notice is that "If you are having difficulty finding a reviewer, please reach out to the [community](https://flink.apache.org/what-is-flink/community/)." Is coming though as-is not as the expected markdown link.
I suggest we try to get the markdown links working properly or leave as just the URL. We should not see the square and round brackets.

Reply:
Ah right, I tested it on a normal [GH comment](https://github.com/apache/flink/pull/25953#issuecomment-2592558519) and saw the behaviour from the Stale PR comment. It is difficult to test this behaviour and I was _hoping_ that it would pass through as markdown and parsed in the same way as a comment. 

However, that issue you found suggests otherwise. I can just remove the markdown and put the raw link in.